manifests/v/vim/vim/8.2.3187/vim.vim.locale.en-US.yaml

```diff
@@ -0,0 +1,24 @@
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.defaultlocale.1.0.0.schema.json
+PackageIdentifier: vim.vim
+PackageVersion: 8.2.3187
+PackageLocale: en-US
+Publisher: Bram Moolenaar et al.
+Author: Bram Moolenaar et al.
+PackageName: Vim
+PackageUrl: http://www.vim.org/
+License: Copyright (C) 1991-2020 Bram Moolenaar [Bram@vim.org] - Charityware / GNU GPL compatible
+LicenseUrl: https://github.com/vim/vim-win32-installer#license--copyright
+ShortDescription: Vim is a highly configurable text editor built to make creating and changing any kind of text very efficient
+Moniker: vim
+Tags:
+- vim
+- gvim
+- vi
+- text editor
+- text editing
+- code editor
+- utility
+- tool
```

Comment:
```suggestion
Tags:
- vim
- g-vim
- vi
- text-editor
- text-editing
- code-editor
- utility
- tool
```

Reply:
I'm happy to do it, but why can't we have spaces in tags?

Reply:
I don't know, but in one of my PR, Levvie suggested to do it because it's the best practice as mentioned in the documentation. Same I have done here.

Reply:
> edit: ignore me, saw your comment in the other PR. will keep in mind, thanks!

Ok